vars.tf

```diff
@@ -0,0 +1,11 @@
+variable "environment" {}
+
+variable "service" {}
+
+variable "cost_centre" {}
+
+variable "owner" {}
+
+variable "created_by" {}
+
+variable "vpc_id" {}
```

Comment:
last line an empty line. git prefers this.

Reply:
done